src/renderers/shaders/ShaderChunk/bsdfs.glsl

```diff
@@ -0,0 +1,149 @@
+float calcLightAttenuation( float lightDistance, float cutoffDistance, float decayExponent ) {
+
+	if ( decayExponent > 0.0 ) {
+
+	  return pow( saturate( -lightDistance / cutoffDistance + 1.0 ), decayExponent );
+
+	}
+
+	return 1.0;
+
+}
+
+
+vec3 BRDF_Diffuse_Lambert( const in vec3 diffuseColor ) {
+
+	// factor of 1/PI in BRDF omitted as incoming light intensity is scaled up by PI because it is considered a punctual light source
+
```

Comment:
That is not saying it correctly. Something like this, instead:

    // factor of 1/pi omitted because it is cancelled by pi in the rendering equation for punctual lights sources

I will fix the inline comments later.